dist/tools/jlink/jlink.sh

```diff
@@ -99,6 +93,8 @@ test_hexfile() {
         echo "Error: Unable to locate HEXFILE"
         echo "       (${HEXFILE})"
         exit 1
+    else
```

Comment:
Do we need to print on success? If so, then `test_elffile` should also be adapted.

Reply:
True, I'd say we remove that part, is not really needed. I'll remove it and squash directly.

Reply:
Done.